docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

```diff
@@ -1239,16 +1242,49 @@ Only one of the following parameter is allowed, also when one of them is configu
 [source,xml]
 ----
 "test_parameters": {
-   "qr-code": "MT:-24J042C00KA0648G00"
-   "manual-code": "34970112332"
+  "qr-code": "MT:-24J042C00KA0648G00"
+  "manual-code": "34970112332"
 }
 ----
 +
 WARNING: This is an invalid configuration. TH will not accept both parameters set at same time.
 
 On completion of the "network" and the "dut_config" configuration, select the *Update* and then *Create* button to create the Test Project. 
 
-===== Upload PICS File
+==== PIXIT Support
```

Comment:
This section is about the same thing as the new `Test Parameters for SDK Python Tests` section (I believe TH only supports PIXIT for python tests, so this section was a way to show how to pass these parameters to those tests). I suggest merging the two sections (maybe just completely substituting this section with the new one would be enough)

Reply:
Moved pixie section